robottelo/ui/base.py

```diff
@@ -539,11 +540,14 @@ def click(self, locator, wait_for_ajax=True,
         :raise: UINoSuchElementError if the element could not be found.
 
         """
-        element = self.wait_until_element(locator, timeout=waiter_timeout)
+        if type(locator).__name__ == 'tuple':
```

Comment:
:+1: 
also, i'd like to suggest the opposite check:
```python
from selenium.webdriver.remote.webelement import WebElement
[...]
        if type(locator) == WebElement:
            element = locator
        else:
            element = self.wait_until_element(locator, timeout=waiter_timeout)
```
For me, it's much more clear of what's going on when we check for WebElement instead of some tuple, but that may be just personal preference... Not a blocker, of course, but please apply @rplevka's suggestion at least.